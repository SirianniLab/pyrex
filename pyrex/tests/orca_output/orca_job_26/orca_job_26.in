!rhf sto-3g 


*xyz 0 1
C    0.193729379050   -0.229205007684    0.056780702928
O   -1.168040234738   -0.253464980226   -0.030300370539
O    1.073906773458    0.585278044604   -0.039288308255
H    0.480214170685   -1.304904205766    0.280942239838
H   -1.292950356545   -1.232105156788    0.147402489587
*


