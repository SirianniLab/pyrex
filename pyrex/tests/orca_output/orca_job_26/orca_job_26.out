
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #               Directorship: Frank Neese             #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 4.2.0 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLED
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM Hessian, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : CASSCF linear response (MC-RPA)
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : AUTO-CI
   Lucas Lang             : DCDCAS
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse,             : VdW corrections, initial TS optimization,
                  C. Bannwarth                     DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Andreas Klamt, Michael Diedenhofen            : otool_cosmo (COSMO solvation model)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, Multilevel, MM, QM/MM, CI optimization
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 4.2.3
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


----- Orbital basis set information -----
Your calculation utilizes the basis: STO-3G
   H-Ne       : W. J. Hehre, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2657 (1969).
   Na-Ar      : W. J. Hehre, R. Ditchfield, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2769 (1970).
   K,Ca,Ga-Kr : W. J. Pietro, B. A. Levy, W. J. Hehre and R. F. Stewart, J. Am. Chem. Soc. 19, 2225 (1980).
   Sc-Zn,Y-Cd : W. J. Pietro and W. J. Hehre, J. Comp. Chem. 4, 241 (1983).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.in
|  1> !rhf sto-3g 
|  2> 
|  3> 
|  4> *xyz 0 1
|  5> C    0.193729379050   -0.229205007684    0.056780702928
|  6> O   -1.168040234738   -0.253464980226   -0.030300370539
|  7> O    1.073906773458    0.585278044604   -0.039288308255
|  8> H    0.480214170685   -1.304904205766    0.280942239838
|  9> H   -1.292950356545   -1.232105156788    0.147402489587
| 10> *
| 11> 
| 12> 
| 13> 
| 14>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  C      0.193729   -0.229205    0.056781
  O     -1.168040   -0.253465   -0.030300
  O      1.073907    0.585278   -0.039288
  H      0.480214   -1.304904    0.280942
  H     -1.292950   -1.232105    0.147402

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 C     6.0000    0    12.011    0.366095   -0.433135    0.107300
   1 O     8.0000    0    15.999   -2.207276   -0.478979   -0.057259
   2 O     8.0000    0    15.999    2.029390    1.106015   -0.074244
   3 H     1.0000    0     1.008    0.907473   -2.465912    0.530904
   4 H     1.0000    0     1.008   -2.443322   -2.328341    0.278550

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     1.364766698253     0.00000000     0.00000000
 O      1   2   0     1.203047855657   137.47271962     0.00000000
 H      1   2   3     1.135539825467   104.32915531   180.01517251
 H      2   1   3     1.002455704861    97.49116259   179.96955735

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     2.579035296394     0.00000000     0.00000000
 O      1   2   0     2.273430973193   137.47271962     0.00000000
 H      1   2   3     2.145859284294   104.32915531   180.01517251
 H      2   1   3     1.894366743574    97.49116259   179.96955735

---------------------
BASIS SET INFORMATION
---------------------
There are 3 groups of distinct atoms

 Group   1 Type C   : 6s3p contracted to 2s1p pattern {33/3}
 Group   2 Type O   : 6s3p contracted to 2s1p pattern {33/3}
 Group   3 Type H   : 3s contracted to 1s pattern {3}

Atom   0C    basis set group =>   1
Atom   1O    basis set group =>   2
Atom   2O    basis set group =>   2
Atom   3H    basis set group =>   3
Atom   4H    basis set group =>   3

Checking for AutoStart:
The File: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.gbw exists
Trying to determine its content:
     ... Fine, the file contains calculation information
     ... Fine, the calculation information was read
     ... Fine, the file contains a basis set
     ... Fine, the basis set was read
     ... Fine, the file contains a geometry
     ... Fine, the geometry was read
     ... Fine, the file contains a set of orbitals
     ... Fine, the orbitals can be read
     => possible old guess file was deleted
     => GBW file was renamed to GES file
     => GES file is set as startup file
     => Guess is set to MORead
     ... now leaving AutoStart

------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...   33
 # of primitive gaussian functions       ...   51
 # of contracted shells                  ...   11
 # of contracted basis functions         ...   17
 Highest angular momentum                ...    1
 Maximum contraction depth               ...    3
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  1.000e-10
 Primitive cut-off              TCut     ...  1.000e-11


------------------------------ INTEGRAL EVALUATION ----------------------------


 * One electron integrals 
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   24
 Basis Dimension        Dim             ....   17
 Nuclear Repulsion      ENuc            ....     68.6218996999 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... LIBINT
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 2.400e-01
Time for diagonalization                   ...    0.003 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.003 sec
Total time needed                          ...    0.006 sec

---------------------
INITIAL GUESS: MOREAD
---------------------
Guess MOs are being read from file: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.ges
Input Geometry matches current geometry (good)
Input basis set matches current basis set (good)
MOs were renormalized
MOs were reorthogonalized (Cholesky)
                      ------------------
                      INITIAL GUESS DONE (   0.0 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
                      *** Initiating the SOSCF procedure ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  0   -186.19129335 -186.1912933515  0.000001  0.000001  0.000002  0.000000
               *** Restarting incremental Fock matrix formation ***
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   1 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :         -186.19129335 Eh           -5066.52267 eV

Components:
Nuclear Repulsion  :           68.62189970 Eh            1867.29682 eV
Electronic Energy  :         -254.81319305 Eh           -6933.81949 eV
One Electron Energy:         -393.03762494 Eh          -10695.09750 eV
Two Electron Energy:          138.22443189 Eh            3761.27801 eV

Virial components:
Potential Energy   :         -370.75298905 Eh          -10088.70173 eV
Kinetic Energy     :          184.56169570 Eh            5022.17906 eV
Virial Ratio       :            2.00882956


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -1.0914e-11  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    2.2991e-06  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    2.5272e-07  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    7.8928e-07  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    1.1709e-06  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.scfp) ****
             **** ENERGY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.scfp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -20.334284      -553.3240 
   1   2.0000     -20.292299      -552.1815 
   2   2.0000     -11.187189      -304.4189 
   3   2.0000      -1.385992       -37.7148 
   4   2.0000      -1.313606       -35.7450 
   5   2.0000      -0.768089       -20.9008 
   6   2.0000      -0.690469       -18.7886 
   7   2.0000      -0.558253       -15.1908 
   8   2.0000      -0.547793       -14.9062 
   9   2.0000      -0.497784       -13.5454 
  10   2.0000      -0.372978       -10.1492 
  11   2.0000      -0.345644        -9.4054 
  12   0.0000       0.301130         8.1942 
  13   0.0000       0.505199        13.7472 
  14   0.0000       0.605807        16.4848 
  15   0.0000       0.720227        19.5984 
  16   0.0000       0.992528        27.0081 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 C :    0.275321
   1 O :   -0.272116
   2 O :   -0.232107
   3 H :    0.012175
   4 H :    0.216726
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 C s       :     3.083975  s :     3.083975
      pz      :     0.908745  p :     2.640703
      px      :     0.792777
      py      :     0.939181
  1 O s       :     3.833192  s :     3.833192
      pz      :     1.853423  p :     4.438924
      px      :     1.270446
      py      :     1.315054
  2 O s       :     3.862816  s :     3.862816
      pz      :     1.244314  p :     4.369291
      px      :     1.502482
      py      :     1.622495
  3 H s       :     0.987825  s :     0.987825
  4 H s       :     0.783274  s :     0.783274


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 C :    0.213617
   1 O :   -0.179566
   2 O :   -0.170356
   3 H :   -0.024140
   4 H :    0.160446

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 C s       :     3.033699  s :     3.033699
      pz      :     0.913394  p :     2.752684
      px      :     0.867682
      py      :     0.971608
  1 O s       :     3.696838  s :     3.696838
      pz      :     1.850656  p :     4.482729
      px      :     1.303413
      py      :     1.328660
  2 O s       :     3.743673  s :     3.743673
      pz      :     1.244298  p :     4.426683
      px      :     1.533765
      py      :     1.648619
  3 H s       :     1.024140  s :     1.024140
  4 H s       :     0.839554  s :     0.839554


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 C      5.7247     6.0000     0.2753     3.9117     3.9117     0.0000
  1 O      8.2721     8.0000    -0.2721     2.1394     2.1394     0.0000
  2 O      8.2321     8.0000    -0.2321     2.1937     2.1937    -0.0000
  3 H      0.9878     1.0000     0.0122     0.9999     0.9999    -0.0000
  4 H      0.7833     1.0000     0.2167     0.9530     0.9530     0.0000

  Mayer bond orders larger than 0.100000
B(  0-C ,  1-O ) :   1.0240 B(  0-C ,  2-O ) :   1.9656 B(  0-C ,  3-H ) :   0.9072 
B(  1-O ,  2-O ) :   0.1554 B(  1-O ,  4-H ) :   0.9260 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.488 sec
Sum of individual times     ....       0.150 sec  ( 30.8%)

Fock matrix formation       ....       0.123 sec  ( 25.1%)
Diagonalization             ....       0.000 sec  (  0.0%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.019 sec  (  3.8%)
Initial guess               ....       0.002 sec  (  0.3%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.0%)
SOSCF solution              ....       0.007 sec  (  1.5%)

-------------------------   --------------------
FINAL SINGLE POINT ENERGY      -186.191293351500
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.gbw
Electron density file                           ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_26/orca_job_26.scfp
The origin for moment calculation is the CENTER OF MASS  = ( 0.000066, -0.000066  0.000017)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:      0.40183       1.43034      -0.24530
Nuclear contribution   :     -0.76395      -2.37519       0.40082
                        -----------------------------------------
Total Dipole Moment    :     -0.36212      -0.94484       0.15552
                        -----------------------------------------
Magnitude (a.u.)       :      1.02374
Magnitude (Debye)      :      2.60215



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:     3.740849     0.346107     0.316797 
Rotational constants in MHz : 112147.823351 10376.028391  9497.326368 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :    -0.686883    -0.759106    -0.000154 
x,y,z [Debye]:    -1.745918    -1.929493    -0.000390 

 

Timings for individual modules:

Sum of individual times         ...        1.021 sec (=   0.017 min)
GTO integral calculation        ...        0.435 sec (=   0.007 min)  42.6 %
SCF iterations                  ...        0.586 sec (=   0.010 min)  57.4 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 1 seconds 358 msec
